allennlp/service/server_sanic.py

```diff
@@ -6,8 +6,8 @@
 
 # Move access.log and error.log to /tmp by default
 # If someone really wants them, they can move them back
-LOGGING['handlers']['accessTimedRotatingFile']['filename'] = '/tmp/sanic_access.log'
-LOGGING['handlers']['errorTimedRotatingFile']['filename'] = '/tmp/sanic_error.log'
+# LOGGING['handlers']['accessTimedRotatingFile']['filename'] = '/tmp/sanic_access.log'
+# LOGGING['handlers']['errorTimedRotatingFile']['filename'] = '/tmp/sanic_error.log'
```

Comment:
@joelgrus fixed the issue that made this necessary.  We can just remove these lines.  The error was because you upgraded sanic - this was the reason sanic was pinned in the first place.